### PR TITLE
Sprint5 merge

### PR DESCRIPTION
**Pull request by Zagoshipda**
Sprint5 resolved merge conflict related with ProblemSetCreate page

캡처

Screenshot 2021-12-06 at 02-59-45 ProbLoom

### CODE REVIEW

**Review by mman39 — APPROVED** [review]

File: frontend/probloom/src/containers/ProblemSet/ProblemSetSearch/ProblemSetSearch.tsx [hunk 1]
```diff
@@ -40,20 +40,122 @@ const creatorOptions = [
...
+      value: 'tag-music',
+    },
+    {
+      key: 'tag-business',
+      text: 'business',
+      value: 'tag-business',
+    },
+  ],
+};
+
```
Comment (mman39) [inline frontend/probloom/src/containers/ProblemSet/ProblemSetSearch/ProblemSetSearch.tsx:158]: I think tags and sub-tags also have to be received from backend. I will change this part after views implemented.

Comment (mman39) [inline frontend/probloom/src/containers/ProblemSet/ProblemSetSearch/ProblemSetSearch.tsx:158]: I think tags and sub-tags also have to be received from backend. I will change this part after views implemented.

Comment (Zagoshipda) [reply frontend/probloom/src/containers/ProblemSet/ProblemSetSearch/ProblemSetSearch.tsx:158]: Checked. Then I will keep this.

**Comment by TLB-MISS** [comment]
Have you not yet implemented frontend and test for userStatistic?

**Comment by Zagoshipda** [comment]
merged with https://github.com/swsnu/swpp2021-team13/pull/71

**Pull request merged**